### PR TITLE
fix(chart): make rbac more explicit

### PR DESCRIPTION
**Pull request by pborn-ionos**

### CODE REVIEW

File: charts/uptrends/templates/cluster_role.yaml [hunk 1]
```diff
@@ -6,12 +6,21 @@ metadata:
...
-- nonResourceURLs:
-  - '*'
+  - get
+  - list
+  - update
+  - watch
+- apiGroups:
+  - operators.ionos-cloud.github.io
+  resources:
+  - uptrends
```
Comment (mcbenjemaa) [inline charts/uptrends/templates/cluster_role.yaml:20]: please add status sub resource and finalizer

File: charts/uptrends/templates/cluster_role.yaml [hunk 2]
```diff
@@ -6,12 +6,21 @@ metadata:
...
   verbs:
-  - '*'
+  - get
```
Comment (mcbenjemaa) [inline charts/uptrends/templates/cluster_role.yaml:25]: we need delete verb also, as part of the ingress deleted event.

**Review by mcbenjemaa — CHANGES_REQUESTED** [review]

File: charts/uptrends/templates/cluster_role.yaml (hunk 2 above)
Comment (pborn-ionos) [reply charts/uptrends/templates/cluster_role.yaml:25]: Good catch. Updated.

File: charts/uptrends/templates/cluster_role.yaml (hunk 1 above)
Comment (pborn-ionos) [reply charts/uptrends/templates/cluster_role.yaml:20]: Updated as discussed.

**Review by mcbenjemaa — APPROVED** [review]
LGTM

**Pull request merged**